src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -25,19 +25,21 @@
 class ResizeFormListener implements EventSubscriberInterface
 {
     protected $type;
-    protected $options;
+    protected $entryOptions;
```

Comment:
because this is protected, its name must be kept
let's also keep the same name on the constructor argument

Reply:
You are totally correct, sorry I didn't think about that. Fixed!